docs/glossary.md

```diff
@@ -259,7 +269,19 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
 #### [Multi-Cluster Replication](/self-hosted-guide/multi-cluster-replication)
 
-Multi-Cluster Replication is a feature which asynchronously replicates Workflow Executions from active Clusters to other passive Clusters, for backup and state reconstruction.
+Multi-Cluster Replication asynchronously replicate Workflow Executions and metadata from an active [Cluster](#temporal-cluster) to a standby Cluster.
+The replicated data is used for backup and state reconstruction.
+
+_Tags: [term](/tags/term), [explanation](/tags/explanation)_
+
+<span style={{ backgroundColor: 'rgb(255, 255, 102)', fontWeight: 'bold' }}>When MRN goes live update this link from cloud to cloud/multi-region</span>
+#### [Multi-region Namespace](/cloud)<!--/multi-region)-->
+
+A Multi-region Namespace (MRN) is a [Temporal Cloud](#temporal-cloud) Namespace that is configured to work across an active [region](/cloud/service-availability#regions) and a standby region located within the same continent.
+Each region is hosted on a separate Temporal Cloud [Cluster](#temporal-cluster). 
```

Comment:
Maybe something like "Each Region corresponds to a dedicated Temporal Cloud Service" -

We are deprecating the use of the term "Cluster" - and even if we were not, I don't think we would want to use it to describe an instance of Temporal Cloud.